gpu/octree/src/cuda/octree_builder.cu

```diff
@@ -245,7 +245,7 @@ namespace pcl
                                     octree.begs [offset + i] = cell_begs[i];
                                     octree.ends [offset + i] = cell_begs[i + 1];
                                     octree.codes[offset + i] = parent_code_shifted + cell_code[i];
-
+                                    octree.nodes[offset + i] = 0;
```

Comment:
@SergioRAgostinho Git blame shows you last worked on this file. Could you please verify the patch? 😄 

I don't know if this will break some behavior apart from correcting the bug.

Reply:
It was a PR spellchecking variables and comments and I was not the original author, I just hit the merge button. The sad truth is that I never really touched any logic in gpu related modules, so I know as much as you do about it, if not less. :)

Reply:
`OctreeGlobal` is undocumented. But the logic feels correct (nodes = empty unless they have a valid task allocated to them)